### PR TITLE
docs: add open source commitment document

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/open-source/commitment.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+sidebar_title: Commitment
+---
+
+Ory is fundamentally rooted in open source, its community dynamics, and development principles. Ory's software is licensed under
+the Apache 2.0 license. Ory's founders have decades of experience in open source and open knowledge.
+
+To build open source software and help its community and adoption grow, Ory offers a commercial product which uses open source
+software from Ory and others like the Linux foundation.
+
+## Open Source is a good idea
```
Comment (kmherrmann) [inline docs/open-source/commitment.mdx:13]:
```suggestion
## Why Ory is developing Open Source```

File: docs/open-source/commitment.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Building a profitable open source business is non-trivial. But there are many good reasons why
+[open source is the right choice](https://opensource.google/documentation/reference/why) for companies like Ory:
+
+- Engineering Economics: Releasing work as open source and the corresponding contribution process eventually result in a higher
+  return on the initial investment made versus the alternative closed source process.
+- Establishing or Supporting an Open Standard: Publishing your project under an open source license can encourage adoption of a
+  standard. When a project becomes a standard, you benefit from a massive influx of outside contributors so the project, and the
+  ecosystem around it, evolves more rapidly.
+- Disintermediation: Releasing open source software can serve to decouple things and create an opening to compete in a new field.
```
Comment (kmherrmann) [inline docs/open-source/commitment.mdx:23]:
```suggestion
- Disintermediation: Releasing open source software can serve to decouple software components and create an opening to compete in a new field.
```

File: docs/open-source/commitment.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  Consider Kubernetes, software for automating management, deployment, operations and scaling of containerized applications.
+- Drive Adoption with a Platform and Ecosystem: Releasing a project as open source allows others to adapt and build on top of your
```
Comment (kmherrmann) [inline docs/open-source/commitment.mdx:25]:
```suggestion
- Driving Adoption of a Platform and Ecosystem: Releasing a project as open source allows others to adapt and build on top of your
```

**Review by kmherrmann — DISMISSED** [review]

**Review by jonas-jonas — COMMENTED** [review]
Great to have this in writing! 🙌🏼

File: docs/open-source/commitment.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+To build open source software and help its community and adoption grow, Ory offers a commercial product which uses open source
+software from Ory and others like the Linux foundation.
+
+## Open Source is a good idea
+
+Building a profitable open source business is non-trivial. But there are many good reasons why
+[open source is the right choice](https://opensource.google/documentation/reference/why) for companies like Ory:
+
+- Engineering Economics: Releasing work as open source and the corresponding contribution process eventually result in a higher
```
[the remark this answers is not in the document]
Comment (jonas-jonas) [reply docs/open-source/commitment.mdx:18]: Ah :D 

Hm, I read it as one item, as one comes from the other. It's a weird sentence... I'm fine with it either way, just wanted to catch a typo. :)

File: docs/open-source/commitment.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+To build open source software and help its community and adoption grow, Ory offers a commercial product which uses open source
+software from Ory and others like the Linux foundation.
+
+## Why Ory is developing Open Source
+
+Building a profitable open source business is non-trivial. But there are many good reasons why
+[open source is the right choice](https://opensource.google/documentation/reference/why) for companies like Ory:
+
+- Engineering Economics: Releasing work as open source and the corresponding contribution process eventually result in a higher
```
Comment (vinckr) [inline docs/open-source/commitment.mdx]:
```suggestion
- Engineering economics: Releasing work as open source and the corresponding contribution process eventually result in a higher
```

Please follow "normal" [capitalization](https://en.wikipedia.org/wiki/Capitalization_in_English) rules 🙏

**Review by vinckr — APPROVED** [review]
I made a few format changes and simplified language in a few places, hope you dont mind ;-)

A good start, we can work on the doc over time, but this should work 👍

**Comment by aeneasr** [comment]
@vinckr https://github.com/ory/docs/pull/1129/commits/87c0bd2cf395337979333f0a6f9dca0ce0464652 actually broke the format, because it's now a bunch of singular list items with paragraphs inbetween. I also think that "open source" is more recognized than OSS (operating system settings?), so let's just use that terminology!

**Comment by vinckr** [comment]
> @vinckr [87c0bd2](https://github.com/ory/docs/commit/87c0bd2cf395337979333f0a6f9dca0ce0464652) actually broke the format, because it's now a bunch of singular list items with paragraphs inbetween. I also think that "open source" is more recognized than OSS (operating system settings?), so let's just use that terminology!

true, the correct solution would have been using h3 or h4... 
Thanks for fixing it 🙏

**Pull request merged**